observability-lib/grafana/dashboard.go

```diff
@@ -17,20 +17,20 @@ const (
 	TypePlatformDocker     TypePlatform = "docker"
 )
 
-type Dashboard struct {
+type Observability struct {
 	Dashboard            *dashboard.Dashboard
 	Alerts               []alerting.Rule
 	ContactPoints        []alerting.ContactPoint
 	NotificationPolicies []alerting.NotificationPolicy
 }
```

Comment:
renaming as the struct is holding all type of grafana resources not only dashboard anymore